packages/components/src/CollapsiblePanel/__snapshots__/CollapsiblePanel.snapshot.test.js.snap

```diff
@@ -529,6 +536,62 @@ exports[`CollapsiblePanel should render default without content 1`] = `
 </div>
 `;
 
+exports[`CollapsiblePanel should render panel with custom element 1`] = `
+<div
+  className="panel panel-default theme-tc-collapsible-panel theme-default-panel"
+>
+  <div
+    className="theme-panel-heading panel-heading"
+  >
+    <button
+      blah="blah"
+      className="theme-panel-title panel-title btn btn-link"
+      disabled={false}
+      onClick={[Function]}
+      type="button"
+    >
+      <div
+        className="theme-panel-title panel-title"
+      >
+        <div
+          className="theme-group theme-col-2"
+        >
+          <span
+            className="theme-undefined"
+            onFocus={[Function]}
+            onMouseOver={[Function]}
+          >
+            Version 1 Version 1
+          </span>
+          <span
+            className="theme-undefined"
+            onFocus={[Function]}
+            onMouseOver={[Function]}
+          >
+            Custom
```

Comment:
Hmm I don't really understand. In you test, the custom element is `<span>Custom element</span>` but the rendered one is not that